src/import_vdf.py

```diff
@@ -104,6 +105,133 @@ def import_pinecone(args):
     pinecone_import.upsert_data()
 
 
+def import_vertexai_vectorsearch(args):
+    """
+    Import data to Vertex AI Vector Search
+    """
+    set_arg_from_input(args, "project_id", "Enter the Google Cloud Project ID:")
+    set_arg_from_input(args, "location", "Enter the region hosting your index: ")
+    set_arg_from_input(
+        args,
+        "target_index_name",
+        "Enter the name of the index to import to:",
+        default_value=None,
+    )
+    set_arg_from_input(
+        args,
+        "batch_size",
+        "Enter size of upsert batches (default: 100):",
+        default_value=100,
+        type_name=int,
+    )
+    set_arg_from_input(
+        args,
+        "filter_restricts",
+        "Optional. Enter list of dicts describing string filters for each data point: ",
+    )
+    set_arg_from_input(
+        args, "numeric_restricts", "Optional. Enter list of dicts for each datapoint: "
+    )
+    set_arg_from_input(
+        args, "crowding_tag", "Optional. CrowdingTag of the datapoint: ", type_name=str
+    )
+    if args["create_new_index"] is True:
+        set_arg_from_input(
+            args,
+            "gcs_bucket",
+            "Required. Valid gcs bucket name: ",
+            type_name=str,
+            # required=True
+        )
```

Comment:
same here. it would be good to show the options, or create a new one if user hits enter

Reply:
got it - if user leaves blank will create new bucket in import class